docs/source/guide/s3-example-mrap.rst

```diff
@@ -0,0 +1,27 @@
+.. Copyright 2010-2025 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Is 2010-2025 correct here? I think we can just use the year this was authored, I don't know if it's required (or correct) to include a date in the past for net-new content. Is this some special requirement of CC BY-SA 4.0?
```suggestion
.. Copyright 2025 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```